azext_iot/common/certops.py

```diff
@@ -125,3 +134,201 @@ def open_certificate(certificate_path: str) -> str:
         raise ValueError("Certificate file type must be either '.pem' or '.cer'.")
     # Remove trailing white space from the certificate content
     return certificate.rstrip()
+
+
+def create_root_certificate(
```

Comment:
can we combine this with create_self_signed_certificate and/or replace it? There is some customization that would need to be added (such as file prefix, etc)

It may also be helpful to add docstrings here as to what this will be used for (ex: create a root certificate that will be uploaded to the iot hub)

Reply:
I've added docstrings to the certificate commands. For the time being, my recommendation would be to leave these as separate calls, as we may iterate on some of the Edge device certificate generation, and until we know we have one function to rule them all (certs) I think we should leave them separate just in case.